.travis.yml

```diff
@@ -1,82 +1,50 @@
 sudo: true
-language: generic
+language: haskell
 
 git:
   depth: 5
 
+cabal: "head"
```

Comment:
Interesting idea! I guess it makes sense to build `summoner` with `head` version of `cabal`. But in scaffolded projects it's probably better to use specific version of cabal, that will be in `.cabal` file as well.

Reply:
Nice catch!